packages/lex_service/src/lexicon/utils/validate-lex-entry.util.ts

```diff
@@ -0,0 +1,18 @@
+import { LexiconEntry } from '../models/lexicon-entry.model';
+import { Lexicon } from '../models/lexicon.model';
+import Ajv from 'ajv';
+import { BadRequestException } from '@nestjs/common';
+
+export function validateLexEntrySchema<T extends Partial<LexiconEntry>>(lexicon: Lexicon, lexiconEntry: T) {
```

Comment:
I'd recommend having this be a method on the `LexiconEntryService` where it can validate just the `fields` field

Reply:
Should be all set now